ui/apps/platform/src/Containers/Policies/Wizard/Step1/PolicyMetadataFormSection.tsx

```diff
@@ -1,160 +1,136 @@
+            <FormLabelGroup
+                label="Severity"
+                fieldId="severity"
+                errors={errors}
+                isRequired
+                touched={touched}
+                helperText={'Select a severity level for this policy'}
+            >
                 <Flex direction={{ default: 'row' }}>
-                    <Field name="severity" type="radio" value="LOW_SEVERITY">
-                        {({ field }) => (
-                            <Radio
-                                name={field.name}
-                                value={field.value}
-                                onChange={(event, _value) => onChange(_value, event)}
-                                label="Low"
-                                id="policy-severity-radio-low"
-                                isChecked={field.checked}
-                            />
-                        )}
-                    </Field>
-                    <Field name="severity" type="radio" value="MEDIUM_SEVERITY">
-                        {({ field }) => (
-                            <Radio
-                                name={field.name}
-                                value={field.value}
-                                onChange={(event, _value) => onChange(_value, event)}
-                                label="Medium"
-                                id="policy-severity-radio-medium"
-                                isChecked={field.checked}
-                            />
-                        )}
-                    </Field>
-                    <Field name="severity" type="radio" value="HIGH_SEVERITY">
-                        {({ field }) => (
-                            <Radio
-                                name={field.name}
-                                value={field.value}
-                                onChange={(event, _value) => onChange(_value, event)}
-                                label="High"
-                                id="policy-severity-radio-high"
-                                isChecked={field.checked}
-                            />
-                        )}
-                    </Field>
-                    <Field name="severity" type="radio" value="CRITICAL_SEVERITY">
-                        {({ field }) => (
-                            <Radio
-                                name={field.name}
-                                value={field.value}
-                                onChange={(event, _value) => onChange(_value, event)}
-                                label="Critical"
-                                id="policy-severity-radio-critical"
-                                isChecked={field.checked}
-                            />
-                        )}
-                    </Field>
+                    <Radio
+                        name="severity"
+                        value="LOW_SEVERITY"
+                        onChange={() => setFieldValue('severity', 'LOW_SEVERITY')}
```

Comment:
Not a blocker for this contribution, but might reduce friction if `handleChange` supports radio input.

Reply:
I like that idea. Created a function for all of these radio buttons.